src/jv_print.c

```diff
@@ -114,13 +114,18 @@ static void put_indent(int n, int flags, FILE* fout, jv* strout, int T) {
 }
 
 static void jvp_dump_string(jv str, int ascii_only, FILE* F, jv* S, int T) {
+  put_char('"', F, S, T);
+  jvp_dump_string_raw(str, ascii_only, F, S, T);
```

Comment:
What happens if the string has double-quote characters in it?

Reply:
Oh, I guess this isn't new...

Reply:
That's actually a good question.. can jvp_dump_string be tricked into producing bad output?